integrated_channels/degreed/client.py

```diff
@@ -13,7 +13,7 @@
 
```

Comment:
I think the changes in Degreed channel can be discarded as we no longer use this code.

Reply:
@sameenfatima78 reverted those changes